views/templates/block.html.twig

```diff
@@ -2,15 +2,15 @@
 
 {% block body %}
 	<div class="home__form mt-5 d-flex justify-content-center">
-		<form action="/custom_cms/block-insert" class = "p-4" method = "POST">
+		<form action="/custom-cms/block-insert" class = "p-4" method = "POST">
```

Comment:
why are we prefixing `/custom-cms` in the path. It should ideally just have the path only.. can you try with base path HTML tag?